caf-audit/src/main/java/com/hpe/caf/auditing/elastic/ElasticAuditConfiguration.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2015-2017 Hewlett Packard Enterprise Development LP.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.auditing.elastic;
+
+import com.hpe.caf.api.Configuration;
+
+import java.util.List;
+
+@Configuration
+public class ElasticAuditConfiguration {
+
+    public ElasticAuditConfiguration() {
+
+    }
+
+    private List<String> hostNames;
+
+    private int port;
+
+    private String clusterName;
+
+    public List<String> getHostNames() {
+        return hostNames;
+    }
+
+    public void setHostnames(List<String> hostNames) {
+        this.hostNames = hostNames;
+    }
+
+    public int getPort() {
```

Comment:
It's likely that different hosts will be listening on different ports. Or one host might have two nodes running on different ports. You need a way to supply a list of hosts and ports. Don't use two lists it will be too error prone.